compiler+runtime/src/cpp/clojure/core_native.cpp

```diff
@@ -702,6 +702,27 @@ jank_object_ptr jank_load_clojure_core_native()
     intern_fn_obj("range", fn);
   }
 
+  {
+    auto const fn(
+      make_box<obj::jit_function>(behavior::callable::build_arity_flags(3, false, false)));
+    fn->arity_0 = []() -> object * {
+      return iterate(__rt_ctx->intern_var("clojure.core", "inc").expect_ok()->deref(), make_box(0));
+    };
+    fn->arity_1 = [](object * const end) -> object * {
+      return obj::integer_range::create(expect_object<obj::integer>(end)->data);
+    };
+    fn->arity_2 = [](object * const start, object * const end) -> object * {
+      return obj::integer_range::create(expect_object<obj::integer>(start)->data,
+                                        expect_object<obj::integer>(end)->data);
+    };
+    fn->arity_3 = [](object * const start, object * const end, object * const step) -> object * {
+      return obj::integer_range::create(expect_object<obj::integer>(start)->data,
+                                        expect_object<obj::integer>(end)->data,
+                                        expect_object<obj::integer>(step)->data);
+    };
+    intern_fn_obj("integer_range", fn);
```

Comment:
When you intern a var here, you're entering Clojure land. In Clojure land, we use kebab-case, not snake_case. Look at all of the other examples in this file.